### ISSUE TITLE
TASKING: CHIATURA - CAS

### ISSUE THREAD

**Issue by 132ndNeck** [132nd-vWing/SDCM#9]
AREA: CHIATURA
TASK: CAS
RESPONSIBLE: NECK

**Comment by 132nd-Entropy** [comment]
tasking should be beefed up imo, the last 2 missions Ive flown it as CAS flight, there was very little action on the ground and almost all targets only static

**Comment by 132ndNeck** [comment]
Yes, I agree. I will have a look once I get back to ensure that this is finished and more activity is included.
